### PR TITLE
verify how skywalking connects to the micrometer

### PR DESCRIPTION
**Pull request by mufiye**
I add sample codes to verify how skywalking connects to the micrometer which is the [issue#11246](https://github.com/apache/dubbo/issues/11246) .

### CODE REVIEW

File: 4-governance/dubbo-samples-spring-boot-tracing-skywalking/dubbo-samples-spring-boot-tracing-skwalking-provider/src/main/resources/application.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    name: dubbo-springboot-skywalking-demo-provider
+
+management:
+  tracing:
+    sampling:
+      probability: 1.0
+
+dubbo:
+  application:
+    name: dubbo-springboot-skywalking-demo-provider
```
Comment (conghuhu) [inline 4-governance/dubbo-samples-spring-boot-tracing-skywalking/dubbo-samples-spring-boot-tracing-skwalking-provider/src/main/resources/application.yml:28]: name: ${spring.application.name}

**Review by conghuhu — COMMENTED** [review]
fix license ci.

**Review by conghuhu — COMMENTED** [review]
Displays the address of the specified skywalking server

**Comment by mufiye** [comment]
> fix license ci.

I will fix it.

**Review by conghuhu — APPROVED** [review]
LGTM  cc @AlbumenJ

**Review by AlbumenJ — APPROVED** [review]
We should try to verify this case in Integration Test in the future

**Pull request merged**